.github/workflows/cla.yml

```diff
@@ -0,0 +1,28 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheckcla' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Alpha Release
+        uses: cla-assistant/github-action@v2.0.1-alpha
+        env:
+          # Generated and maintained by github
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # JFrog organization secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.CLA_SIGN_TOKEN }}
+        with:
+          path-to-signatures: 'signed_clas.json'
+          path-to-cla-document: 'https://jfrog.com/privacy-policy/cla/'
+          remote-organization-name: 'jfrog'
+          remote-repository-name: 'jfrog-signed-clas'
+          # branch should not be protected
+          branch: 'master'
```

Comment:
@RobiNino Thanks for the PR  :) This Github action needs to be also on `pre-unified-platform` branch 
Can you please check if this can be configured on mutiple branches like master and pre-unified-platform branch

Reply:
Hi @chukka , This action will be triggered on every PR to any branch on this repository. The branch you see specified in the workflow yml is the branch to write to in the destination repo, where we keep the signed CLAs.